problems/objects.js

```diff
@@ -215,7 +303,20 @@ const mostCommonElement = () => {};
  * @returns {string[]} Elements or their pair values.
  */
 
-const updateList = () => {};
+const updateList = (pairs, arr) => {
```

Comment:
so for this one, if we know how to to make an object that counts the occurrences, think about how we can use that to find the most common element